libmamba/include/mamba/core/history.hpp

```diff
@@ -38,6 +38,7 @@ namespace mamba
             static UserRequest prefilled(const Context& context);
 
             std::string date;
+            int revision_num;
```

Comment:
```suggestion
            int revision_num = 0;
```
Always (well, almost always) set a default value for `int` and `float` members as otherwise they will be undefined and will lead to (potentially invisible) undefined behavior if used before being set. (could be also another value, like `-1` if that helps catching incorrect usage)